src/wallet/api.rs

```diff
@@ -1309,4 +1309,9 @@ impl Wallet {
 
         Ok(descriptors_to_import)
     }
+
+    pub fn get_new_address(&mut self) -> Result<Address, WalletError> {
+        let new_address = self.get_next_external_address()?;
+        Ok(new_address)
+    }
```

Comment:
Why do we need this function instead of just calling `get_next_external_address`?

Reply:
Yes please. Redundant functions are maintenance burdens.

Reply:
> Yes please. Redundant functions are maintenance burdens.

done [here](https://github.com/citadel-tech/coinswap/compare/884beab96b35536793dfaecc24e8981f76975990..4246b2575f150e3f57c56af58246733b0a5e515d)